app/helpers/mappings_helper.rb

```diff
@@ -14,16 +13,17 @@ module MappingsHelper
   #    )
   def bootstrap_flavour_tabs(titles_to_links, options)
     content_tag :ul, class: 'nav nav-tabs' do
-      titles_to_links.inject('') do |result, title_link|
-        title, href       = title_link[0], title_link[1]
+      titles_to_links.inject('') { |result, title_link|
+        title = title_link[0]
+        href = title_link[1]
         active            = options[:active] == title
         html_opts         = {}
         html_opts[:class] = 'active' if active
 
         result << content_tag(:li, html_opts) do
           link_to(title, active ? '#' : href)
         end
-      end.html_safe
+      }.html_safe
```

Comment:
Yeah, there's a cop about not chaining on `do...end` blocks, and there were a few of these to fix up.  I don't know if there's an alternative way - but I'll see if anything can be re-arranged.